debian/buster/foreman/rules

```diff
@@ -28,8 +31,8 @@ build:
 
 override_dh_installinit:
 	dh_installinit
-	dh_installinit --name=dynflowd
 
 override_dh_systemd_enable:
 	dh_systemd_enable
-	dh_systemd_enable --name=dynflowd
```

Comment:
While making the change, perhaps keep dynflowd in for a bit so we can build it in nightly and test the installer PR? Then have a simple drop PR like in https://github.com/theforeman/foreman-packaging/pull/4442.

Reply:
OK :) sounds reasonable. Should I keep enabling both services, or just `dynflowd` then?

Reply:
I think this implementation is correct for now and allows us to test it.